scripts/pre-safety-checks.sh-example

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# If this script exists, it is called before the local safety checks, including checking 
+# if there are local services running on required ports.
+
+# This example forcebily shuts down systemd-resolved, and keeps it down, removing the
+# need to prompt the user to shut it down while running.
+
+sudo systemctl disable systemd-resolved.service
```

Comment:
I wouldn't recommend disabling any system services, stopping should be enough (same goes for not enable in post-flash.sh-example).  It's one thing to require a user to change system settings as a per-requesite, but it's bad practice to change system settings within (even as an example).

Reply:
That specific line was required on my system for some reason, without it about 50% of the time `resolved` would start again (I'm guessing from a DNS request) before the cloudcutter service took over port 53.

Reply:
That's going to vary system to system, which is why CloudCutter tries to remain pretty minimal, but we don't want default scripts, even if examples, potentially breaking systems.

My recommendation would be to comment it out, even in the example script, and add a comment explaining the ramifications of using it.